util/chunk/row_container.go

```diff
@@ -229,6 +251,12 @@ func (a *SpillDiskAction) Action(t *memory.Tracker) {
 		logutil.BgLogger().Info("memory exceeds quota, spill to disk now.",
 			zap.Int64("consumed", t.BytesConsumed()), zap.Int64("quota", t.GetBytesLimit()))
 	})
+	if atomic.LoadUint32(&a.c.readOnly) == 1 {
+		err := a.c.spillToDisk()
+		if err != nil {
+			panic(err)
```

Comment:
Should we change `Action(t *Tracker)` to `Action(t *Tracker) error`

Reply:
Should we ignore and log the error and try to find the next action ?